### PR TITLE
Add (WIP) Footer

### PR DESCRIPTION
**Pull request by a0ax**
Adds a WIP footer to the layout. Also makes the logo wordmark link to the homepage.

Desktop:
image

Mobile:
image

### CODE REVIEW

**Comment by Szedann** [comment]
🤔 I'm not sure if rabbitminers is the correct copyright holder

**Comment by Szedann** [comment]
also on mobile the link section seems off-center
image

**Comment by Szedann** [comment]
anyways, looks very good other than those small issues

**Comment by a0ax** [comment]
All easily changed, I put Rabbitminers as the copyright holder because in the LICENSE file he is listed as the copyright holder.
You're right that it's off center; there's some asymmetric padding. It can be fixed by removing the style highlighted below.
image
image

**Comment by Szedann** [comment]
I noticed you added a `w-fit-content` class, I think that's that the `w-fit` class does
https://github.com/Create-Schematics/Create-Schematics/blob/d1dfb4f8a82e061deb6343494f00151a5d6ad09b/frontend/src/app.css#L38-L40

**Comment by a0ax** [comment]
That it does! This is my first time doing anything serious with tailwind so i'm reliant on their docs and wasn't able to find that. Thanks for the catch.

File: frontend/src/routes/+layout.svelte [hunk 1]
```diff
@@ -33,5 +33,41 @@
...
+    <div class="container mx-auto text-center md:text-left p-4 opacity-100 ">
+      <div class="grid grid-cols-1 md:grid-cols-2 gap-4 max-w-6xl mx-auto">
+
+        <div class="mb-4 md:mb-0 items-center">
+          <div class="w-fit mx-auto">
+            <h2 class="text-lg font-semibold mb-2">Create Schematics</h2>
+            <p class="text-m">
+              Create Schematics is <a href="https://github.com/Create-Schematics/Create-Schematics">open source</a>.<br>
+              <a href="https://discord.gg/GJsQadv9Mc">Discord</a><br>
+              <a href="emailto:contact@createschematics.com">Email</a>
```
Comment (Szedann) [inline frontend/src/routes/+layout.svelte]: this url should probably start with `mailto:`, not `emailto:`

**Comment by Szedann** [comment]
LGTM

File: frontend/src/routes/+layout.svelte (hunk 1 above)
Comment (a0ax) [reply frontend/src/routes/+layout.svelte]: Definitely; oversight by me.

**Pull request merged**